lumicks/pylake/detail/units.py

```diff
@@ -0,0 +1,37 @@
+
+au = "dimensionless"
+
+
+def determine_unit(unit1, unit2, operation):
+    """Compares units. Raises exception when units are not easily converted or incompatible.
+
+    Parameters
+    ----------
+    unit1 : string
```

Comment:
`str` is the canonical type name in Python.

Reply:
Replaced 👍